examples/dts/dts_activity_sequence.py

```diff
@@ -0,0 +1,71 @@
+import os
+from azure.identity import DefaultAzureCredential
+
+"""End-to-end sample that demonstrates how to configure an orchestrator
+that calls an activity function in a sequence and prints the outputs."""
+from durabletask import client, task, worker
+from durabletask.accessTokenManager import AccessTokenManager
```

Comment:
Per Python naming conventions, we should use snake_case in module names rather than camelCase. For example, `access_token_manager` instead of `accessTokenManager`. Please see how we're naming all the other files in this repo.